app/presenters/finder_presenter.rb

```diff
@@ -21,61 +21,21 @@ class FinderPresenter
            :no_index?,
            :canonical_link?,
            :all_content_finder?,
-           :eu_exit_finder?, to: :content_item
+           :eu_exit_finder?,
+           :title,
+           :base_path,
+           :government?,
+           :government_content_section,
+           :organisations, to: :content_item
 
 
   def initialize(content_item, facets, values = {})
     @content_item = content_item
-    @organisations = content_item.links.fetch("organisations", [])
-    @values = values
     @facets = facets
     @keywords = values["keywords"].presence
   end
 
-  def name
-    content_item.title
-  end
-
-  def slug
-    content_item.base_path
-  end
```

Comment:
👍 👍 👍 
I'm used to a "slug" being a single part of the path, not the full path.  `slug` meaning `base_path` was very confusing.